app/services/file_manager/file_upload/file_upload.py

```diff
@@ -338,7 +338,6 @@ def resume_get_unfinished_items(
                 ):
                     SrvErrorHandler.customized_handle(
                         ECustomizedError.INVALID_RESUMABLE_FILE_SIZE,
-                        if_exit=True,
```

Comment:
Did you use this variable in any place?

Reply:
oh not really. By default is false so it report error but not exit.

Reply:
Understand.